dialog/base_test.go

```diff
@@ -0,0 +1,29 @@
+package dialog_test
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/dialog"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/test"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+
+	"testing"
+)
+
+func TestDialog_Background_Group(t *testing.T) {
+	a := test.NewApp()
+	defer test.NewApp()
+	test.ApplyTheme(t, theme.LightTheme())
+
+	w := a.NewWindow("")
```

Comment:
Apologies I realise now, the software painter is not wired up by default due to a circular dependency we are working to resolve. You'll need to create a window this way in the meantime:

```
	test.NewApp().Settings().SetTheme(theme.LightTheme())
	defer test.NewApp()
	w := test.NewWindowWithPainter(canvas.NewRectangle(theme.BackgroundColor()), software.NewPainter())
```